build-tools/installers/create-installers.targets

```
@@ -0,0 +1,122 @@
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\Xamarin.Android.NUnitLite.dll"></BCLFrameworkFiles>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\android-support-multidex.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\cil-strip.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\class-parse.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\desugar_deploy.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\FSharp.Compiler.CodeDom.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\FSharp.Core.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\generator.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\illinkanalyzer.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Irony.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\javadoc-to-mdoc.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Export.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Cecil.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Diagnostics.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.JavaCallableWrappers.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Runtime.Environment.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Runtime.Environment.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\jcw-gen.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\jnimarshalmethod-gen.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\LayoutBinding.cs"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\logcat-parse.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mdoc.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle-api.h"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mono-symbolicate.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CSharp.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Options.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Terminal.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\monodoc.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\MULTIDEX_JAR_LICENSE"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\proguard.jar"></MSBuildFiles>
```

I think this and the following line should avoid the `**` and just use `proguard` explicitly:

```suggestion
    <MSBuildFiles Include="$(MSBuildSrcDir)\proguard\lib\proguard.jar" />
    <MSBuildFiles Include="$(MSBuildSrcDir)\proguard\license.html" />
```